core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -159,6 +159,23 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
     }
   }
 
+  it should "copy entities and attributes to new workspace for clone" in withDefaultTestDatabase {
```

Comment:
I poked around a bit and what I see is the important test is further down, here in `EntityComponentSpec`: "clone all entities from a workspace containing cycles". Note that test uses `copyAllEntities()` so you'd have to update it, unless you take over that method name.

If you decide to keep this test too, it would be good to verify the contents of the entities against each other, instead of just checking the counts. Doing that would make the "We should clone attributes as well as entities" assertion obselete.